DBZ-4812 Add Db2 platform related config (#5598)
debezium/debezium: documentation/modules/ROOT/pages/connectors/db2.adoc

The Db2 connector has been tested with Db2 for Linux.
It is expected that the connector would also work on other platforms such as Windows,
and we'd love to get your feedback if you can confirm this to be the case.

> The Db2 connector has also *incubating* support for Db2 on z/OS.

Reviewer: ```suggestion
You can also use the Db2 connector with Db2 for z/OS.
This feature is currently  *incubating* and is subject to change in future releases.
```

|Specifies the time, in milliseconds, that the connector waits for a query to complete.
Set the value to `0` (zero) to remove the timeout limit.

ifdef::community[]
|[[db2-property-cdc-control-schema]]<<db2-property-cdc-control-schema, `cdc.control.schema`>>
|`ASNCDC`
|Specifies the name of the database schema where CDC control structures are located.

|[[db2-property-cdc-change-tables-schema]]<<db2-property-cdc-change-tables-schema, `cdc.change.tables.schema`>>
|`ASNCDC`

> |Specifies the name of the database schema where CDC change tables are located.

Reviewer: ```suggestion
|Specifies the name of the database schema that stores CDC control structures.
```
